### PR TITLE
Add reverse thrust (brake) channel to quadplane mixer files

### PR DESCRIPTION
**Pull request by ThomasRigi**
**Describe your solution**
Added reverse thrust (brake) channel to quadplane mixer files vtol_AAERT.aux.mix and vtol_AAVVT.aux.mix. Updated description of the corresponding airframes to include the reverse thrust channel.

I copy-pasted the corresponding lines from the deltaquad mixer file.

**Describe possible alternatives**

**Test data / coverage**
Still untested on the new mixer files. I currently don't have access to a quadplane on which I could test it. I might be able to test it on a Generic AAVVT v-tail plane airframe with Quad VTOL (SYS_AUTOSTART = 13007) in the next month. 

**Additional context**
PR to tackle first point of https://discuss.px4.io/t/quadplane-back-transition-improvements/15054
@sfuhrer @RomanBapst

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/airframes/13000_generic_vtol_standard
Comment (sfuhrer) [inline ROMFS/px4fmu_common/init.d/airframes/13000_generic_vtol_standard]: I would prefer to call it "Throttle reverse" - as it also can be a puller and not a pusher. You can also drop the "channel" here.

**Review by sfuhrer — CHANGES_REQUESTED** [review]
Thanks for your PR - functionality-wise this looks good. We should though then rename the mixer files, eg from AAERT_quad to AAERTB_quad (where the "B" would stand for "Breaking" - other propositions welcome!)

**Comment by ThomasRigi** [comment]
I just had an aha-moment concerning the names of the mixer files :sweat_smile: I now see that they need to be renamed. B for braking sounds good to me.

Also, I should tag the maintainers @SimonWilks  @AndreasAntener and @sanderux to make sure they're ok with the changes. Otherwise I could leave the existing mixers unchanged and create new mixers files with the braking, leaving the Fun Cub, the QuadRanger and the SparkleTech Ranger airframes untouched.

File: ROMFS/px4fmu_common/init.d/airframes/13000_generic_vtol_standard
Comment (ThomasRigi) [reply ROMFS/px4fmu_common/init.d/airframes/13000_generic_vtol_standard]: Proposition accepted. Will change on the next commit.

**Comment by sfuhrer** [comment]
> I just had an aha-moment concerning the names of the mixer files sweat_smile I now see that they need to be renamed. B for braking sounds good to me.
> 
> Also, I should tag the maintainers @SimonWilks @AndreasAntener and @sanderux to make sure they're ok with the changes. Otherwise I could leave the existing mixers unchanged and create new mixers files with the braking, leaving the Fun Cub, the QuadRanger and the SparkleTech Ranger airframes untouched.

I would prefer to not having new ones just for the reverted throttle, but let's see if anyone has a different opinion.

**Comment by sanderux** [comment]
I see no issue as it is assigning an unused channel. Unfortunately, we have retired our quadranger.

**Comment by ThomasRigi** [comment]
@sfuhrer can you please check that I renamed all necessary files?

**Comment by sfuhrer** [comment]
@ThomasRigi I've looked through your changes and took the liberty to push a commit fixing some minor naming/commenting directly to your branch. Please have a look if you agree with what I did, otherwise this is good to be merged from my side.

**Comment by ThomasRigi** [comment]
@sfuhrer Good changes, thanks! Please go ahead and merge it :+1:

**Review by sfuhrer — DISMISSED** [review]

**Comment by dagar** [comment]
This should be good to merge once the conflicts are resolved.

**Comment by sfuhrer** [comment]
> This should be good to merge once the conflicts are resolved.

That would be the case now.

**Comment by sfuhrer** [comment]
@ThomasRigi we have discussed this PR in last week's devcall, and came to the conclusion that it is would be a bit dangerous to bring this in, as it would mess up people's setup if they were using some of the AUX channels for camera/gimbal or other payload controls. And at the same time we are not aware of many people using the pusher as a breaker as well. So we concluded that we rather do not want to bring this in atm. 
Are you yourself blocked anyhow in not having it upstreamed? Or do you know if people wanted to use the reverse before, but in the end didn't because it was not supported for their airframe?

**Comment by ThomasRigi** [comment]
@sfuhrer Thanks for letting me know. I initially thought that all these AUX channel assignments would need to be done in the mixer files, so that the people who have a more advanced setup would automatically have their own mixer file. Through experience I have learned that you can overwrite some AUX channels directly via parameters, for example the camera trigger signal which defaults to channels 5&6 (which is a great way to accidentally check if your pusher motor spins without any preflight checks directly after reboot :grimacing: ). So yeah, I can see your concern.

With https://github.com/PX4/Firmware/pull/14405 I hope that our need for using the pusher to brake will be greatly reduced. By next week we should have finished the build of the next iteration of our testing platform and be ready to get some testing done. In the ideal case reversing the pusher thrust will have become obsolete.

**Comment by ThomasRigi** [comment]
In case we'd still like / want / need the reverse thrust in upstream, would it be possible to create a new airframe that contains this mixer so that the others are unaffected?

In general, what are the conditions that need to be met for a new airframe to be integrated in upstream PX4?

**Comment by sfuhrer** [comment]
@ThomasRigi Learning by doing at its finest :grin: 

Yes absolutely! There are no strict criteria for upstreaming an airframe, but in general I would say it should be either something that can be purchased (at least in a similar configuration), or something that's held general (like adding the reverse to one of the existing general VTOLs). We just don't want to create airframes that we don't know if ever somebody uses them. 
That's how I see it, so please feel free to create a new airframe if you have something flying.

In the meantime lets close this PR, okay?

**Pull request closed without merging**
